src/client/app/utils/determineCompatibleUnits.ts

```diff
@@ -314,7 +337,7 @@ function groupCase(currentUnits: Set<number>, newUnits: Set<number>, defaultGrap
  * @param compatibilityChangeCase Which GroupCase is involved.
  * @returns the desired color for styling.
  */
-function getMenuOptionFont(compatibilityChangeCase: GroupCase): React.CSSProperties {
+export function getMenuOptionFont(compatibilityChangeCase: GroupCase): React.CSSProperties {
```

Comment:
This function does not seem to be used outside this file so the added export is not needed. I prefer not to export items not used when possible.